### PR TITLE
Dev -> Master for v1.1 release

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
LGTM! :rocket:

File: docs/usage.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ This downloads a text file called `SRR_Acc_List.txt` which can be directly provi
 
 The typical command for running the pipeline is as follows:
 
-```bash
+```console
 nextflow run nf-core/fetchngs --input ids.txt -profile docker
 ```
```
Comment (ewels) [inline docs/usage.md:43]: I'm a little surprised that the markdown linting isn't failing for this. I've hit linting errors before due to it.

I think that if it's a one-line without a prompt, it should be `bash`. If it's multi-line with output and a prompt, use `console`.

So, bash:

```bash
nextflow run nf-core/fetchngs --input ids.txt -profile docker
```

Console:

```console
$ nextflow run nf-core/fetchngs --input ids.txt -profile docker

Some output
Much wow
Isn't this command amazing?
```

I think that this is purely stylistic - the syntax highlighting etc will likely be identical.

File: nextflow_schema.json [hunk 2]
```diff
@@ -29,6 +29,12 @@
                     "description": "Comma-separated list of ENA metadata fields to fetch before downloading data.",
                     "help_text": "The default list of fields used by the pipeline can be found at the top of the [`bin/sra_ids_to_runinfo.py`](https://github.com/nf-core/fetchngs/blob/master/bin/sra_ids_to_runinfo.py) script within the pipeline repo. This pipeline requires a minimal set of fields to download FastQ files i.e. `'run_accession,experiment_accession,library_layout,fastq_ftp,fastq_md5'`. Full list of accepted metadata fields can be obtained from the [ENA API](https://www.ebi.ac.uk/ena/portal/api/returnFields?dataPortal=ena&format=tsv&result=read_run])."
                 },
+                "sample_mapping_fields": {
+                    "type": "string",
+                    "fa_icon": "fas fa-globe-americas",
+                    "description": "Comma-separated list of ENA metadata fields used to create mappings file for sample id mapping or renaming.",
```
Comment (ewels) [inline nextflow_schema.json:35]: No mention of the MultiQC config file here.

**Review by ewels — APPROVED** [review]
Couple of super minor / trivial comments, but generally LGTM! 🚀

File: docs/usage.md (hunk 1 above)
Comment (JoseEspinosa) [reply docs/usage.md:43]: Good to know

Comment (drpatelh) [reply docs/usage.md:43]: Ah, yes. Makes sense. I have always just used them interchaneably.

Comment (drpatelh) [reply docs/usage.md:43]: FIxed https://github.com/nf-core/fetchngs/pull/16/commits/ea7bd2f77e86db1158e8294a150e6befcdc09d10

File: nextflow_schema.json (hunk 2 above)
Comment (drpatelh) [reply nextflow_schema.json:35]: Added https://github.com/nf-core/fetchngs/pull/16/commits/8f03fa7fc741e12b129643361fdeab43569413f7

**Pull request merged**